docs/src/examples.rst

```diff
@@ -14,205 +14,175 @@
-         tableName: 'MyAppTable',
-         readCapacity: 5,
-         writeCapacity: 5
-       });
+.. _creating_ecs_l2_example_2:
 
-       table.addPartitionKey({ name: 'Alias', type: dynamodb.AttributeType.String });
-       table.addSortKey({ name: 'Timestamp', type: dynamodb.AttributeType.String });
-     }
-   }
+Step 2: Add the |EC2| and |ECS| Packages
+----------------------------------------
 
-   const app = new cdk.App();
+Install support for |EC2| and |ECS|.
 
-   new MyStack(app, 'MyStack');
+.. tabs::
 
-    app.run();
+    .. group-tab:: TypeScript
 
-.. _creating_rds_example:
+        .. code-block:: sh
 
-Creating an |RDS| Database
-==========================
+            npm install @aws-cdk/aws-ec2 @aws-cdk/aws-ecs
 
-The following example creates the Aurora database **MyAuroraDatabase**.
+.. _creating_ecs_l2_example_3:
 
-.. code-block:: js
+Step 3: Create a Fargate Service
+--------------------------------
 
-   import ec2 = require('@aws-cdk/aws-ec2');
-   import rds = require('@aws-cdk/aws-rds');
-   import cdk = require('@aws-cdk/cdk');
+There are two different ways of creating a serverless infrastructure with |ECS|:
 
-   class MyStack extends cdk.Stack {
-     constructor(parent: cdk.App, name: string, props?: cdk.StackProps) {
-       super(parent, name, props);
+- Using the **Fargate** launch type, where |ECS| manages your cluster resources
+- Using the **EC2** launch type, where you manage your cluster resources
 
-       const vpc = new ec2.VpcNetwork(this, 'VPC');
+This example creates a Fargate service,
+which requires a VPC, a cluster, a task definition, and a security group.
+Later on we'll show you how to launch an EC2 service.
 
-       new rds.DatabaseCluster(this, 'MyRdsDb', {
-         defaultDatabaseName: 'MyAuroraDatabase',
-         masterUser: {
-           username: 'admin',
-           password: '123456'
-         },
-         engine: rds.DatabaseClusterEngine.Aurora,
-         instanceProps: {
-           instanceType: new ec2.InstanceTypePair(ec2.InstanceClass.Burstable2, ec2.InstanceSize.Small),
-           vpc: vpc,
-           vpcPlacement: {
-             subnetsToUse: ec2.SubnetType.Public
-           }
-         }
-       });
-     }
-   }
+.. tabs::
 
-   const app = new cdk.App();
+    .. group-tab:: TypeScript
 
-   new MyStack(app, 'MyStack');
+        Add the following import statements:
 
-   app.run();
+        .. code-block:: typescript
 
-.. _creating_s3_example:
+            import ec2 = require('@aws-cdk/aws-ec2');
+            import ecs = require('@aws-cdk/aws-ecs');
 
-Creating an |S3| Bucket
-=======================
+        Add the following code to the end of the constructor:
 
-The following example creates the |S3| bucket **MyS3Bucket** with server-side KMS
-encryption provided by |S3|.
+        .. code-block:: typescript
 
-.. code-block:: js
+            const vpc = new ec2.VpcNetwork(this, 'MyVpc', {
+              maxAZs: 2 // Default is all AZs in region
```

Comment:
3 is actually a better default if we're going to teach people best practices. 2 is just for tests.

Reply:
ok